fitwsarah_frontend/src/views/PersonalTrainerPanel/TrainerAppointments.js

```diff
@@ -178,6 +178,76 @@ function AdminAccounts() {
             });
     };
 
+    const getAppointmentInfo = async (appointmentId) => {
+        try {
+            const response = await fetch(`${process.env.REACT_APP_BASE_URL}/api/v1/appointments/${appointmentId}`, {
+                method: "GET",
+                headers: new Headers({
+                    Authorization: "Bearer " + accessToken,
+                    "Content-Type": "application/json"
+                })
+            });
+    
+            if (!response.ok) {
+                throw new Error('Failed to get Appointment Info');
+            }
+    
+            const data = await response.json();
+            console.log(data)
+            getAccountEmail(data.time,data.date,data.userId);
+        } catch (error) {
+            console.error(error);
+        }
+    };
+    
+    const getAccountEmail = async (time, date, userId) => {
+        try {
+            const response = await fetch(`${process.env.REACT_APP_BASE_URL}/api/v1/accounts/users/${userId}`, {
+                method: "GET",
+                headers: new Headers({
+                    Authorization: "Bearer " + accessToken,
+                    "Content-Type": "application/json"
+                })
+            });
+    
+            if (!response.ok) {
+                throw new Error('Failed to get account email');
+            }
+    
+            const data = await response.json();
+            sendCancelMail(time, date, data.email);
+        } catch (error) {
+            console.error(error);
+        }
+    };
+    
+    
+    const sendCancelMail = (time, date, email) => {
+        const cancelMail = {subject: "Cancelled Appointment", message:`Hello, your appointment at ${time} for ${date} has been cancelled.`};
```

Comment:
Can you elaborate on what the subject does in this situation for the const cancelMail?

Reply:
The subject is the header of the email sent.